src/main/kotlin/com/terraformation/backend/nursery/db/NurseryBatchEventData.kt

```diff
@@ -0,0 +1,11 @@
+package com.terraformation.backend.nursery.db
```

Comment:
should this be under the `model` package?

Reply:
I can certainly move it there! that would make more sense

Reply:
oh don't move it yet, if we have similar db models that aren't really persisted, this should be fine.